build.gradle

```diff
@@ -53,9 +53,10 @@ spotless {
   }
 
   // clang-format is difficult to configure across all of our platforms so we
-  // avoid being really strict about enforcing it in our build for now.
+  // avoid being really strict about enforcing it in our build for now. version
+  // 17.x has a bug, so skip that version.
   def clangFormatVersion = getClangFormatVersion()
-  if (!clangFormatVersion.equals('')) {
+  if (!clangFormatVersion.equals('') && clangFormatVersion.indexOf('17') < 0) {
```

Comment:
nit: 18.0.17 would be flagged.

Reply:
yes, then any formatting issues would be caught once the build version increments.